tests/pom.xml

```diff
@@ -74,6 +74,12 @@
     </dependency>
     <dependency>
       <groupId>org.tachyonproject</groupId>
+      <artifactId>tachyon-examples</artifactId>
```

Comment:
I see now, we use the clone MapReduce job. In general it is useful to be able to run example code in integration tests, so disregard my previous comment.